packages/p2p-v2/src/components/Modals/AvailableP2PBalanceModal/__tests__/AvailableP2PBalanceModal.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import AvailableP2PBalanceModal from '../AvailableP2PBalanceModal';
+
+const MockApp = () => {
+    const [isModalOpen, setIsModalOpen] = useState(true);
+
+    return (
+        <>
+            <div id='v2_modal_root' />
+            <AvailableP2PBalanceModal isModalOpen={isModalOpen} onRequestClose={() => setIsModalOpen(false)} />
```

Comment:
can you render just the modal and pass the props directly to it? you can just mock the onRequestClose and check if it has been called 👀